docs/mp/health/01_introduction.adoc

```diff
@@ -26,42 +26,53 @@
 :common-deps-page-prefix-inc: ../../shared/dependencies/common_shared.adoc
 :microprofile-bundle: true
 
+== ToC
+
+- <<Overview, Overview>>
+- <<Maven Coordinates, Maven Coordinates>>
+- <<Usage, Usage>>
+- <<API, API>>
+- <<Configuration, Configuration>>
+- <<Reference, Reference>>
+- <<Examples, Examples>>
+- <<Additional information, Additional Information>>
+
+== Overview
+Microservices expose their health status primarily so external tools (for example, an orchestrator such as Kubernetes)
+can monitor each service and take action, such as restarting a service instance if it has failed
+or temporarily shunting traffic away from the instance if the service is unable to process
+incoming requests normally.
+
 include::{common-deps-page-prefix-inc}[tag=maven-dependency]
 
 [source,xml]
 ----
-        <dependency>
-            <groupId>io.helidon.microprofile.health</groupId>
-            <artifactId>helidon-microprofile-health</artifactId>
-        </dependency>
+<dependency>
+    <groupId>io.helidon.microprofile.health</groupId>
+    <artifactId>helidon-microprofile-health</artifactId>
+</dependency>
 ----
 
 To enable built-in health checks add the following dependency
 (or use the <<mp/introduction/02_microprofile.adoc, helidon-microprofile bundle>> )
 
 [source,xml]
 ----
-        <dependency>
-            <groupId>io.helidon.health</groupId>
-            <artifactId>helidon-health-checks</artifactId>
-        </dependency>
+<dependency>
+    <groupId>io.helidon.health</groupId>
+    <artifactId>helidon-health-checks</artifactId>
+</dependency>
 ----
 
-== Overview
-Microservices expose their health status primarily so external tools (for example, an orchestrator such as Kubernetes)
-can monitor each service and take action, such as restarting a service instance if it has failed
-or temporarily shunting traffic away from the instance if the service is unable to process
-incoming requests normally.
 
-== About the MicroProfile Health Specification
-Helidon MP implements the MicroProfile Health
-link:http://download.eclipse.org/microprofile/microprofile-health-{health-release}/microprofile-health-spec.html[spec].
+== Usage
+
+Helidon MP Health implements MicroProfile Health Checks
+link:http://download.eclipse.org/microprofile/microprofile-health-{health-release}/microprofile-health-spec.html[Specification].
```

Comment:
Link is dead. Also, is it better to link to pdf (not html).

Reply:
Switched to PDF